styles/sass/base/_link.scss

```diff
@@ -0,0 +1,42 @@
+// NAVIGATION LINK
+
+a {
+  color: var(--primary-blue);
+  font-weight: 500;
+  text-decoration: none;
+  position: relative;
+}
+
+a::after {
+  content: '';
+  position: absolute;
+  background-color: var(--primary-blue);
+
+  left: 0;
+  bottom: -4px;
+
+  height: 1px;
+  width: 24px;
+
+  transition-property: width;
+  transition-duration: 0.3s;
+  transition-timing-function: ease-out;
+}
+
+a:hover {
+  color: var(--hover-blue);
+}
+
+a:hover::after {
+  width: 100%;
+  color: var(--hover-blue);
+  background-color: var(--hover-blue);
+}
+
+.lang a::after {
+  width: 8px;
+}
+
+.lang a:hover::after {
+  width: 100%;
+}
```

Comment:
.lang specific should be moved into Navigation.astro, this ain't global style :D